src/pages/StaffLogin/StaffLogin.tsx

```diff
@@ -0,0 +1,16 @@
+import { FC } from 'react'
+import AuthLayout from '@/layouts/AuthLayout/AuthLayout'
+import Login from './components/Login'
+import ThirdPartyLogin from './components/ThirdPartyLogin/ThirdPartyLogin'
+
+const StaffLogin: FC = () => (
```

Comment:
ReNaming and The login page is not vertically centered. Please test it on a widescreen display.